src/components/button/button.tsx

```diff
@@ -129,16 +154,29 @@ export class Button implements LabelableComponent, InteractiveComponent, FormOwn
     }
   }
 
+  @Watch("intlLoading")
+  @Watch("defaultMessages")
+  @Watch("messageOverrides")
+  onMessagesChange(): void {
+    /** referred in t9n util */
+  }
+
   //--------------------------------------------------------------------------
   //
   //  Lifecycle
   //
   //--------------------------------------------------------------------------
 
-  connectedCallback(): void {
+  async connectedCallback(): Promise<void> {
+    if (Build.isBrowser) {
+      connectLocalized(this);
+      connectMessages(this);
+    }
+
+    connectLabel(this);
```

Comment:
If moving this line was not necessary, please make sure label behavior works. Otherwise, you can restore this.